src/service/poc_lora.proto

```diff
@@ -0,0 +1,123 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+import "data_rate.proto";
+
+enum invalid_participant_side {
+  beaconer = 0;
+  witness = 1;
+}
+
+enum invalid_reason {
+  none = 0;
+  bad_signature = 1;
+  not_asserted = 2;
+  entropy_expired = 3;
+  bad_entropy = 4;
+  invalid_capability = 5;
+  invalid_pubkey = 6;
+  irregular_interval = 7;
+  gateway_not_found = 8;
+  denied = 9;
+
+}
+
+// beacon report as submitted by gateway to ingestor
+message lora_beacon_report_req_v1 {
+  bytes beacon_id =
+      1; // likely be a concatenation of local and remote entropy so maybe not
+         // needed if we already include both fields ?
+  bytes pub_key = 2;
+  bytes local_entropy = 3;
+  bytes remote_entropy = 4;
+  bytes data = 5;
+  float frequency = 6;
+  int32 channel = 7;
+  data_rate datarate = 8;
+  int32 tx_power = 9;
+  // Timestamp of beacon report submit
+  uint64 timestamp = 10;
+  bytes signature = 11;
+}
+
+// response returned to gateway submitting beacon report to ingestor
+message lora_beacon_report_resp_v1 { string id = 1; }
+
+// witness report as submitted by gateway to ingestor
+message lora_witness_report_req_v1 {
+  bytes beacon_id = 1;
+  bytes pub_key = 2;
+  bytes packet = 3;
+  uint64 timestamp = 4;
+  uint32 ts_res = 5;
+  sint32 signal = 6;
+  float snr = 7;
+  float frequency = 8;
+  int32 channel = 9;
+  data_rate datarate = 10;
+  bytes signature = 11;
+}
+
+// response returned to gateway submitting witness report to ingestor
+message lora_witness_report_resp_v1 { string id = 1; }
+
+// beacon report outputted by ingestor, tagged with received timestamp
+message lora_beacon_ingest_report_v1 {
+  uint64 received_timestamp = 1;
+  lora_beacon_report_req_v1 report = 2;
```

Comment:
Should stick to one naming convention imo. So like:
`lora_beacon_report_req_v1 beacon_report` or `lora_beacon_report_req_v1 beacon`
`lora_witness_report_req_v1 witness_report` or `lora_witness_report_req_v1 witness`

throughout the proto

Reply:
updated to use 'report': https://github.com/helium/proto/pull/154/commits/a7bd1184cf9e289dfb8aee7cbf865f1ece05d77d